frontend/src/components/App.js

```diff
@@ -109,12 +115,11 @@ const MENU_ITEMS = [
     icon: <ProjectOutlined />,
     text: "Projects",
   },
-  // DISABLED UNTIL THIS FEATURE IS READY TO SHIP
-  // {
-  //   url: "/lab-work",
-  //   icon: <ExperimentOutlined />,
-  //   text: "Lab Work"
-  // },
+  {
+    url: "/lab-work",
+    icon: <ExperimentOutlined />,
+    text: "Lab Work"
```

Comment:
Given the new paradigm we entered, i would move Lab Work to the top of the menu and move Project just below.